client/lib/user/utils.js

```diff
@@ -10,6 +10,16 @@ var debug = require( 'debug' )( 'calypso:user:utilities' ),
 var user = require( 'lib/user' )();
 
 var userUtils = {
+	getLoginUrl: function( redirect ) {
+		let url = config( 'login_url' );
+
+		if ( redirect ) {
+			url += '?redirect_to=' + encodeURIComponent( redirect );
```

Comment:
Is there any chance login_url will ever already have a `?` in it, or that it might in the future?